### DISCUSSION TITLE
Lorem Ipsum // kantlipsum

### DISCUSSION THREAD

**Discussion by edzob** [JRBliekendaal/master-thesis#36] [Ideas]
Hi.. recently bumped into kantlipsum (latex package) :)

**Comment by JRBliekendaal** [comment]
and what does that do extra? I already make use of the lipsum package btw to fill some blanks so I have a good overview of the structure (the visual side of things).

**Comment by edzob** [comment]
It is just a fun alternative 

description from CPAN: https://ctan.org/pkg/kantlipsum?lang=en
The package spits out sentences in Kantian style; the text is provided by the Kant generator for Python by Mark Pilgrim, described in the book “Dive into Python”.

Example output from: https://newbedev.com/alternative-to-lorem-ipsum
"As any dedicated reader can clearly see, the Ideal of practical reason is a representation of, as far as I know, the things in themselves; as I have shown elsewhere, the phenomena should only be used as a canon for our understanding. The paralogisms of practical reason are what first give rise to the architectonic of practical reason. As will easily be shown in the next section, reason would thereby be made to contradict, in view of these considerations, the Ideal of practical reason, yet the manifold depends on the phenomena. Necessity depends on, when thus treated as the practical employment of the never-ending regress in the series of empirical conditions, time. Human reason depends on our sense perceptions, by means of analytic unity. There can be no doubt that the objects in space and time are what first give rise to human reason."
